packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -19,10 +19,16 @@ export enum AutoUpdaterEvents {
 export const defaultUpdateInterval = 12 * 60 * 60 * 1000;
 export const initialUpdateStartup = 5 * 1000;
 const releasesLink = 'https://objects.joplinusercontent.com/r/releases';
+const supportedPlatformAssets: { [key in string]: string } = {
+	'darwin': 'latest-mac.yml',
+	'win32': 'latest.yml',
+};
```

Comment:
How is the support for the Apple ARM64 version? I don't see it mentioned here? In the release it would be named for example "Joplin-3.1.4-arm64.DMG"